### ISSUE TITLE
Multi monitor mode with Ubuntu client and Windows 7 Ultimate host

### ISSUE THREAD

**Issue by vuchl** [FreeRDP/FreeRDP#4262]
Hello there!

I was making my waythrough all the `/multimon` threads and issues for a while now but I can not for the life of me figure out what's going wrong.

I have a Ubuntu 17.10 x64 client and a Windows 7 x64 Ultimate VM. when I connect via
```bash
$ xfreerdp --version
This is FreeRDP version 1.1.0-beta1

$ xfreerdp /monitor-list
* [0] 1920x1080    +0+0
  [1] 1920x1080     +1920+0
```

either of those produced the same result:
spans my 2 monitors on the client over one big on the server

```bash
$ xfreerdp /v:... /u:... /d:... /multimon
$ xfreerdp /v:... /u:... /d:... /multimon:force
$ xfreerdp /v:... /u:... /d:... /multimon /monitors:1,0
$ xfreerdp /v:... /u:... /d:... /multimon:force /monitors:1,0
$ xfreerdp /v:... /u:... /d:... /multimon /monitors:0,1
$ xfreerdp /v:... /u:... /d:... /multimon:force /monitors:0,1
```

So I added the nightly repo as mentioned here: https://ci.freerdp.com/job/freerdp-nightly-binaries/ and installed the v2 nightly:

```bash
$ sudo apt-get install freerdp2-x11

$ xfreerdp --version
This is FreeRDP version 2.0.0-dev (git n/a)
```

Restarting the client with the same CLI options as metioned above is still not working.

So I pulled the `dev` branch, installed the dependencies and built the client and still no multi monitor support.
```bash
$ git clone https://github.com/FreeRDP/FreeRDP FreeRDP-master

$ sudo apt-get install build-essential git-core cmake xsltproc libssl-dev libx11-dev \ 
       libxext-dev libxinerama-dev libxcursor-dev libxdamage-dev libxv-dev libxkbfile-dev \
       libasound2-dev libcups2-dev libxml2 libxml2-dev libxrandr-dev libxi-dev \
       libgstreamer-plugins-base1.0-dev libavutil-dev libavcodec-dev

# I did not install libgstreamer0.10-dev libgstreamer-plugins-base0.10-dev since they arent 
# available and CMAKE did not complain when using the 1.0 version

$ cd FreeRDP-master

$ cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_SSE2=ON .

$ make

$ sudo make install

$ xfreerdp --version
This is FreeRDP version 2.0.0-dev (785c42f46)
```

Still the same.

I read that there is problems with window managers, so I installed GNOME via
```bash
$ sudo apt-get install ubuntu-gnome-desktop
```
After a reboot and logging in via GNOME I still can not use `/multimon` properly. Any ideas?

**Edit:**
I also cloned the potetially openbox compatible repo from https://github.com/RangeeGmbH/FreePDF and built this version. And I installed openbox
```bash
$ git clone https://github.com/RangeeGmbH/FreePDF FreePDF-openbox

$ cd FreePDF-openbox

$ cmake -DCMAKE_BUILD_TYPE=Release -DWITH_SSE2=ON .

$ make

$ sudo make intall

$ xfreerdp --version
This is FreeRDP version 2.0.0-dev (d5344c339)

$ sudo apt-get install openconf obconf

$ xfreerdp /v:... /u:... /d:... /multimon
$ xfreerdp /v:... /u:... /d:... /multimon:force
$ xfreerdp /v:... /u:... /d:... /multimon /monitors:1,0
$ xfreerdp /v:... /u:... /d:... /multimon:force /monitors:1,0
$ xfreerdp /v:... /u:... /d:... /multimon /monitors:0,1
$ xfreerdp /v:... /u:... /d:... /multimon:force /monitors:0,1x
```

This even gives me a streched window but only on one monitor. 

We are trying to get rid of physical windows workstations so a dual monitor setup is prefered and attempted in the VM aswell. But the stretch mode is not ideal.

**Comment by akallabeth** [comment]
@vuchl That is interesting, so have you confirmed the windows 7 VM is correctly working with multi monitor RDP?
Can confim that multimon works with a VM and windows 7 ultimate SP1 from a debian stretch gnome 
```xfreerdp /v:<host> /u:<user> /p:<pwd> /multimon /rfx```

**Comment by hardening** [comment]
Interesting that's almost my test case when testing multi-monitor changes. Except that I'm on Ubuntu 16.04 and it's a Seven x64 host (not ultimate).

**Comment by vuchl** [comment]
Hello @hardening and @akallabeth !

I just tried again and I made big mistake on my end: I was connectiong to a Windows 7 x64 Professional VM. I got a twist in the IP Adress. With areal Windows 7 Ultimate VM it works like a charm.

I just tested it with the `freerdp-x11` and `freerdp2-x11` installtions. So it absolutely does matter which version ist installed.Maybe I will try https://github.com/stascorp/rdpwrap to see if it may open up the RDP server. But I am actually not sure if I am even allowed to do so.

Thanks for the feedback guys and sorry for the unneeded hassle ✌️
